bedrock/externalpages/templates/externalpages/pocket/pocket-hits-signup.html

```diff
@@ -0,0 +1,61 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% block page_title %}Hits Newsletter Signup{% endblock %}
+
+{% block pocket_css %}
+  {{ css_bundle('pocket-hits-signup') }}
+{% endblock  %}
+
+{% block content %}
+{% include 'externalpages/pocket/includes/nav.html' %}
+<div class="mzp-l-content mzp-t-content-lg">
+    <h1 class="pocket-hits-header">Your guide to the most fascinating articles from top publications.</h1>
+    <h2 class="pocket-hits-subheader">Join millions of Pocket Hits subscribers and get inspired with our daily newsletter.</h2>
+    <form class="pocket-hits-signup-form top" id="pocket-hits-signup-top" autocomplete="off">
+        <label for="email-signup-input-top">Your email address</label>
+        <input type="email" placeholder="yourname@example.com" id="email-signup-input-top" name="pocket-hits-email-submission" value=""/>
```

Comment:
**suggestion:** I think we can remove the `value` attribute here? I don't think it's necessary because it's empty anyway. Goes for both this and the input field at the bottom of the page